content/articles/getting-started-with-cypress.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Getting Started With Cypress"
+date: 2019-11-07
+draft: true
+type: "post"
+authors: ["buonincontri"]
+tags: [""]
+lede: "Trying out Cypress for Integration testing"
```

Comment:
Can you flesh this lede to a proper sentence, instead of a clause? You have 30 to 40 words for this. Also, is it "Cypress for Integration" or "integration testing"? The capital "I" makes it look like the "integration" is part of Cypress and not testing.